src/components/pages/devices/flyouts/deviceDetails.js

```diff
@@ -9,17 +9,41 @@ import {
   FlyoutContent
 } from 'components/shared';
 
+import './deviceDetails/deviceDetails.css';
```

Comment:
See the comment in the `deviceDetailsTemp.js` file about the import ordering pattern.